content/docs/user-guide/experiment-management/managing-experiments.md

```diff
@@ -0,0 +1,68 @@
+# Managing Experiments
+
+Once you have defined and/or [run experiments] in your project, you can use
+several features of DVC to see, compare, reproduce, share, or remove them.
+
+[run experiments]: /doc/user-guide/experiment-management/running-experiments
+
+## Experiment names
+
+Experiments created with `dvc exp run` will have an auto-generated name like
```

Comment:
Extracted the whole Experiment names section into a new doc per @dberenbaum's suggestion.

This is temporary though, probably won't be included in the PR for merge... ⏳

Reply:
Need to add to the index if we are keeping this section.

Reply:
I'll make a separate PR for this ⏳